src/tools/schema/common.ts

```diff
@@ -1,7 +1,7 @@
 import {outdent} from 'outdent'
 import {z} from 'zod'
 
-export const DEFAULT_SCHEMA_ID = 'sanity.workspace.schema.default'
+export const DEFAULT_SCHEMA_ID = 'sanity.workspace.schemas.default'
```

Comment:
Wrong id, you want `_.schemas.default` here

In detail:

```ts
export type SchemaId = `_.schemas.${string}`
export type TaggedSchemaId = `_.schemas.${string}.tag.${string}`

`_.schemas.<workspaceName>`
`_.schemas.<workspaceName>.tag.<tag-from --tag tagName in  sanity schema deploy> `

```

Also note: `'default' `workspace name happens when:
* sanity.config.ts does not provide a `name`, or it is literally `name: default`

We have been discussing that Agent Actions should make `schemaId` optional, and if it is missing, look for a single non-tagged schema in the dataset. If it is there, use it. This is a todo though.

Reply:
Btw where did you get this new string from? Id i accidentaly say this was it?

Reply:
My mistake, I misread